apps/web/app/components/dashboard/dashboard-ui.css

```
@@ -0,0 +1,623 @@
+.sidebar:not(.collapsed) .sidebar-text {
+  position: static;
+  visibility: visible;
+  transition: opacity 0.15s ease 0.08s;
+}
+
+.sidebar.collapsed .sidebar-text {
+  position: absolute;
+  width: 0;
+  height: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)